### ISSUE TITLE
Is it possible to check the contents of a tuple on return without a complex validator?

### ISSUE THREAD

**Issue by jeking3** [uzumaxy/pyvalid#10]
This does not seem to work:

``` /tmp/foo.py
from pyvalid import returns

@returns((dict, [str]))
def good():
    return ({"foo": "bar"}, ["42"])

@returns((dict, [str]))
def good2():
    return (None, ["42"])

@returns((dict, [str]))
def good3():
    return (None, None)

good()
good2()
good3()
```

```
root@3794c8d02dc5:/data/blueprints# python3 /tmp/foo.py
Traceback (most recent call last):
  File "/tmp/foo.py", line 15, in <module>
    good()
  File "/usr/local/lib/python3.7/dist-packages/pyvalid/__returns.py", line 39, in decorator_wrapper
    raise InvalidReturnType(type(returns_val), func.__name__)
pyvalid.__exceptions.InvalidReturnType: Invalid return type <class 'tuple'> for good()
```

**Comment by uzumaxy** [comment]
Hey @jeking3 ,
I think the #17 might be a solution. I expect to integrate this functionality into the pyvalid package so it will be possible to perform such checks.
Thanks!

**Comment by uzumaxy** [comment]
Hey @jeking3 ,
It should be implemented in the [0.96 version](https://github.com/uzumaxy/pyvalid/releases/tag/0.96). Here's a link to the related PR: #17
